flash/tabular/classification/model.py

```diff
@@ -83,7 +83,11 @@ def __init__(
 
     def forward(self, x_in) -> torch.Tensor:
         # TabNet takes single input, x_in is composed of (categorical, numerical)
-        x = torch.cat([x for x in x_in if x.numel()], dim=1)
```

Comment:
why not list comprehension ?

Reply:
Jit script is a bit fragile, doesn't support list comprehension (at least not with an `if` in)